packages/app/src/domain/topical/article-list.tsx

```diff
@@ -0,0 +1,76 @@
+import css from '@styled-system/css';
+import styled from 'styled-components';
+import ArrowIcon from '~/assets/arrow.svg';
+import { ArticleLink, ArticleSummary } from '~/components-styled/article-link';
+import { Box } from '~/components-styled/base';
+import { LinkWithIcon } from '~/components-styled/link-with-icon';
+import { Tile } from '~/components-styled/tile';
+import { Heading, Text } from '~/components-styled/typography';
+import siteText from '~/locale';
+import { asResponsiveArray } from '~/style/utils';
+import { useBreakpoints } from '~/utils/useBreakpoints';
+
+type ArticleListProps = {
+  articleSummaries: ArticleSummary[];
+  hideLink?: boolean;
+};
+
+export function ArticleList({ articleSummaries, hideLink }: ArticleListProps) {
+  const breakpoints = useBreakpoints();
+
+  return (
+    <Tile>
+      <Heading level={2}>
+        {siteText.nationaal_actueel.latest_articles.title}
+      </Heading>
+      <Box
+        display="flex"
+        alignItems={{ _: undefined, lg: 'flex-end' }}
+        flexDirection={{ _: 'column', lg: 'row' }}
+      >
+        <Box flex={{ _: undefined, lg: '0 0 33%' }}>
+          <Text m={0}>
+            {siteText.nationaal_actueel.latest_articles.subtitle}
+          </Text>
+        </Box>
+        <Box
+          flex={{ _: undefined, lg: '0 0 66%' }}
+          alignContent={{ _: undefined, lg: 'flex-end' }}
+          justifyContent={{ _: undefined, lg: 'flex-end' }}
+          display={{ _: undefined, lg: 'flex' }}
+          mt={{ _: 2, lg: undefined }}
+        >
+          {!hideLink && (
+            <LinkWithIcon
+              href="/artikelen"
+              icon={<ArrowIcon />}
+              iconPlacement="right"
+            >
+              {siteText.nationaal_actueel.latest_articles.all_articles}
+            </LinkWithIcon>
+          )}
+        </Box>
+      </Box>
+      <Box
+        display="flex"
+        spacingHorizontal={breakpoints.lg}
+        flexDirection={{ _: 'column', lg: 'row' }}
+        flexWrap="wrap"
+      >
+        {articleSummaries.map((summary) => (
+          <ArticleBox key={summary.slug.current}>
+            <ArticleLink articleSummary={summary} />
+          </ArticleBox>
+        ))}
+      </Box>
+    </Tile>
+  );
+}
+
+const ArticleBox = styled(Box)(
```

Comment:
not necessary to extend the Box here
```suggestion
const ArticleBox = styled.div(
```